internal/config/config.go

```diff
@@ -0,0 +1,31 @@
+/*
+SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and cf-service-operator contributors
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"github.com/caarlos0/env/v11"
+)
+
+// Config defines the configuration keys
+type Config struct {
+
+	//Resource cache is enabled or disabled
+	//TODO change it back to false after testing
```

Comment:
Do you want to enable or disable it by default?

Reply:
In the final version, the cache will be disabled by default.
We do not want to change the default behavior.